dist/styles/base.scss

```diff
@@ -373,4 +374,5 @@
 .fr_group_skipped {
   margin-top: $fr_rhythm / 2;
   color: $fr_muted_color;
+  clear: both;
```

Comment:
This puts the "You skipped this message" message beneath the field label, instead of floating it (which puts it beside the label, since the label is also floating).